components/NavBar/navbar.module.css

```diff
@@ -1,99 +1,189 @@
-.navBar {
-  display: flex;
-  background-color: #041484;
-  font-size: 0.9rem;
-  width: 100%;
-  /* padding: 0.05rem 0; */
-  position: relative;
-  align-items: center;
-  justify-content: space-between;
-  margin-bottom: 50px;
+.wrapper {
+  font: 700 16px 'Roboto', sans-serif;
+  margin-bottom: 25px;
 }
-.navBar > ul {
-  display: flex;
-  align-items: center;
-  padding: 0;
-  margin: 0;
+
+.navBar li a {
+  margin: 10px;
+  display: block;
+  border-radius: 0.5rem;
+  color: #ffffff;
+  text-align: center;
+  padding: 19px 16px;
+  text-decoration: none;
 }
+
 .navBar li {
-  list-style-type: none;
-  color: #f2f2f2;
-  font-size: 20px;
-  margin-right: 3rem;
-  height: 100%;
+  float: left;
 }
 
 .navBar li a:hover {
-  color: #e30464;
+  color: #49a82e;
 }
-.logo {
-  margin: 10px 0 5px 10px;
-  cursor: pointer;
+
+.navBarMenu {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #1d1283;
+}
+
+.navBarLogoLi {
+  padding: 14px 20px;
+}
+
+.navBar li.navBarLogoLi a {
+  margin: 0px;
+  padding: 0px;
+}
+
+.navBar li.navBarLoginLi {
+  float: right;
+}
+
+.navBar li.navBarLoginLi a {
+  padding: 16px;
```

Comment:
Can we please make it `12px` ? Sorry for trouble but these minor things will make things more aligned and centered.

Reply:
Yes, there were issues in alignment. Thanks for pointing it out.
Changes are implemented.